src/Elements/Geometry/Vector3.cs

```diff
@@ -503,20 +503,18 @@ public static bool AreCoplanar(this IList<Vector3> points)
         /// <returns></returns>
         public static Transform ToTransform(this IList<Vector3> points)
         {
-            var a = new Vector3(points[1].X - points[0].X,
-                                points[1].Y - points[0].Y,
-                                points[1].Z - points[0].Z).Unitized();
-
+            //var a = new Vector3(points[1].X - points[0].X,
```

Comment:
Delete this commented code.

Reply:
This is a nice cleanup. This code was probably written before we could subtract vectors.